### ISSUE TITLE
Does this actually works ? 

### ISSUE THREAD

**Issue by camiloh95** [rollingstone80/simple-gradient-descent#1]
Parameters vector calculated using the gradient descent algorithm:
[ [ 1084019.9999999998 ],
  [ 4866372.589045913 ],
  [ -808376.8011081647 ],
  [ 254423.31526551873 ],
  [ 4643.968352940207 ],
  [ -350079.43249842536 ] ]

Parameters vector calculated using the normal equation method:
[ [ 1084020 ],
  [ 4866375.931917759 ],
  [ -808378.4699930662 ],
  [ 254422.70107600235 ],
  [ 4643.930267622738 ],
  [ -350079.33140940237 ] ]

Value of minimized cost function: 26332949166

I'm having this results but I don't really understand what they really mean. I would like you give me some rope here.

Thanks by the way for this great example !

**Comment by rollingstone80** [comment]
Hi Camilo! The results are the values of the parameters vector to be applied to the chosen features (in this case sqm, rooms, bathrooms, floor and year of construction), using two different methods (gradient descent and normal equation)

So, say you want to estimate the value of a house in Milan and you have a house with these values:

sqm: 120
rooms: 3
bathrooms: 2
floor: 3
year of construction: 2000

you would first start by scaling the values using your training set and the formula: (raw value - mean of training set) / (max of training set - min of training set) and add the value "1" for the intercept. The resulting scaled values are (rounded to five decimals):

intercept: 1
sqm: -0,05237
rooms: -0,06167
bathrooms: 0,02667
floor: 0,06857
year of construction: 0,31443

You then multiply this vector by the parameters vector and the result is the estimated value in Euros of the house, which in this example would be: €776.033.

The trick is to remember to scale the values before performing the final calculation...
